geopy/geocoders/__init__.py

```diff
@@ -139,7 +141,8 @@
     "what3words": What3Words,
     "yandex": Yandex,
     "ignfrance": IGNFrance,
-    "photon": Photon
+    "photon": Photon,
+    "googleplaces": GooglePlaces
```

Comment:
here is another example of the trailing comma, if photon had already the trailing comma, you wouldn't have to modify that line.